aws-rds-cfn-common/src/main/java/software/amazon/rds/common/handler/HandlerConfig.java

```diff
@@ -19,4 +19,8 @@ public class HandlerConfig {
             .delay(Duration.ofSeconds(30))
             .timeout(Duration.ofMinutes(90))
             .build();
+
+    @Getter
+    @Builder.Default
+    final private int stabilizationInSeconds = 60;
```

Comment:
This looks foreign to the common library. I believe we need a different level of indirection to address this point. As far as I understand, the major reason to have it in the common config is to skip it in the tests. This could be achieved in multiple ways:
- A resource can introduce a local override of HandlerConfig and add the fields that are required.
- The `sleep/1` can be called conditionally, we can use `execOnce` guard for this purpose. In this case the timeout can be hardcoded.
- This config can be a property of the corresponding handler class. It should be pretty trivial to modify it upon a class construction, especially in the tests.

Frankly, I would prefer 1 of these approaches to getting the resource-specific config in this common class as it opens up a Pandora's box and ultimately accumulates random attributes and becomes a dusty corner. Let's avoid it now.

Reply:
Thanks Oleg! Went with first choose to implement DBClusterParameterGroup of implementing local override of HandlerConfig.